main.nf

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env nextflow
 
 /*
-================================================================================
```

Comment:
replacing the `=` line with a `#` line because it interfere when there is merge conflicts, but maybe a `-` line will be more aesthetically pleasing